composer.json

```diff
@@ -25,7 +25,7 @@
         "ext-ctype": "*",
         "doctrine/collections": "^1.5",
         "doctrine/common": "^3.3",
-        "doctrine/dbal": "^3.3",
+        "doctrine/dbal": "^3.3 || 4.0.x-dev",
```

Comment:
We shouldn't declare compatibility here just yet. The new CI job should work without this change.

Reply:
This is for testing. I’ll revert it before the merge.